src/app/static/src/app/components/projects/ProjectHistory.vue

```diff
@@ -143,11 +159,11 @@
             <input type="text"
                    class="form-control"
                    v-translate:placeholder="'projectName'"
-                   @keyup.enter="confirmPromotion(newProjectName)"
+                   @keyup.enter.prevent="confirmPromotion(newProjectName)"
```

Comment:
Could you clarify this - is it a change in behaviour, and if so why? Sorry - not obvious to me! Also differs from pattern used in L185.